ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -360,9 +360,16 @@ def narrow(input, dim, start, length):
     return input[tuple(slices)]
 
 
+@to_ivy_arrays_and_back
+def select_scatter(input, src, dim, index):
+    output = ivy.zeros_like(input)
+    output[dim, index] = src
+    return output
+

```

Comment:
Hey @sanju4725, you'll also need to fix lint/formatting issues in your implementation. Try using `pre-commit` for auto-formatting and running any formatting checks before committing your work. Thanks

Reply:
I have made the necessary changes sir
Please review it again

Reply:
@sanju4725 Ask for a re-review once you've done making all the requested changes on this PR. Thanks